components/elevation/src/vwc-elevation.scss

```diff
@@ -19,12 +19,18 @@ $elevation-border-radius: --elevation-border-radius;
 
 @each $dp in $dps {
 	#{get-default-selector($dps, $dp)} {
-		background-color: var(--vvd-color-surface-#{$dp}dp);
-		filter: var(--vvd-shadow-surface-#{$dp}dp);
+		::slotted(*) {
+			background-color: var(--vvd-color-surface-#{$dp}dp);
+			filter: var(--vvd-shadow-surface-#{$dp}dp);
```

Comment:
rule of thumb - no hardcoded variables

Reply:
I know - but in this option it is not optional to add it because of the scss var inside the css variable. Its the same as in vivid-3:

```
@mixin get-elevation($type) {
	background-color: var(#{--vvd-color-surface-#{$type}dp});
	filter: var(#{--vvd-shadow-surface-#{$type}dp});
}

```